lib/event_store/storage/appender.ex

```diff
@@ -110,9 +112,14 @@ defmodule EventStore.Storage.Appender do
     params = [stream_id_or_uuid, event_count] ++ build_insert_parameters(events)
 
     case Postgrex.query(conn, statement, params, opts) do
-      {:ok, %Postgrex.Result{num_rows: 0}} -> {:error, :not_found}
-      {:ok, %Postgrex.Result{}} -> :ok
-      {:error, error} -> handle_error(error)
+      {:ok, %Postgrex.Result{num_rows: 0}} ->
```

Comment:
This branch can no longer be hit, and whatever case it was guarding against will change. There is no test that was covering this.